src/components/ContactImportationModal/ContactImportationCard/index.jsx

```diff
@@ -30,7 +30,13 @@ const ContactImportationCard = props => {
           <span className="importation-status">{t(statusKey(status))}</span>
         )
       )}
-      {progress && <progress value={progress.current} max={progress.total} />}
+      {progress && (
+        <progress
+          value={progress.current}
+          max={progress.total}
+          className={'importation-progress'}
```

Comment:
Do you now if this works too? 

```suggestion
          className={'u-m-auto u-mv-0'}
```

It would avoid adding special class

Reply:
Thank you very much! Didn't know about this one (mv-0)